environment.rb

```diff
@@ -0,0 +1,90 @@
+module Environment
+  class Depot
+    attr_accessor :packs
+
+    def initialize
+      @packs = {
+        simple_transportation_pack: {
+          :intelligence => [:cellphone],
+          :items => [:medipack, :chevy_versa]
+        },
+        standard_transportation_pack: {
+          :intelligence => [:cellphone, :antenna],
+          :arsenal => [:colt_1911],
+          :items => [:handcuffs, :medipack, :chemistry, :chevy_versa]
+        },
+        simple_mission_pack: {
+          :intelligence => [:infopack, :laptop, :cellphone, :antenna],
+          :arsenal=> [:colt_1911],
+          :items => [:handcuffs, :medipack, :chemistry, :financial]
+        },
+        standard_mission_pack:{
+          :intelligence => [:infopack, :laptop, :cellphone, :antenna],
+          :arsenal => [:remington_870, :colt_1911, :machete, :hatchet],
+          :items => [:handcuffs, :medipack, :chemistry, :financial]
+        }
+      }
+    end
+  end
+  
+  class Control
+    attr_accessor :missions
+
+    def initialize
+      @missions = {}
+    end
+    
+    def new_mission(name: name, objective: objective, pack: pack)
```

Comment:
Method defninitions would be better without parenthesis. 
`def new_mission name: name, objective: objective, pack: pack`